packages/web/src/components/organisms/AuthenticateForm/index.tsx

```diff
@@ -0,0 +1,97 @@
+import React, { useState, useEffect, useCallback } from 'react'
+import Link from 'next/link'
+import { Select } from 'antd'
+import { Form, Input, Button } from 'antd'
+import { useCreateProperty } from 'src/fixtures/dev-kit/hooks'
+
+interface Props {
+  walletAddress: string
+}
+
+const { Option } = Select
+
+const PropertyCreateForm = ({ walletAddress }: Props) => {
+  const [form] = Form.useForm()
+  useEffect(() => {
+    form.setFieldsValue({
+      author: walletAddress
+    })
+  }, [form, walletAddress])
+  const { createProperty } = useCreateProperty()
+  const handleCreateProperty = useCallback(
+    (name: string, symbol: string, author: string) => createProperty(name, symbol, author),
+    [createProperty]
+  )
+
+  const onFinishFailed = (errorInfo: any) => {
+    console.log('Failed:', errorInfo)
+  }
+
+  return (
+    <div style={{ marginTop: '18px' }}>
+      <Form
+        labelCol={{ span: 6 }}
+        wrapperCol={{ span: 18 }}
+        form={form}
+        name="basic"
+        initialValues={{ remember: true }}
+        onFinish={({ name, symbol, author }) => {
+          handleCreateProperty(name, symbol, author)
+        }}
+        onFinishFailed={onFinishFailed}
+      >
+        <Form.Item label="Name" name="name" rules={[{ required: true, message: 'Please input name.' }]}>
+          <Input placeholder="Name" />
+        </Form.Item>
+        <Form.Item label="Symbol" name="symbol" rules={[{ required: true, message: 'Please input symbol.' }]}>
+          <Input placeholder="Symbol" />
+        </Form.Item>
+        <Form.Item label="Author" name="author">
+          <Input type="text"></Input>
```

Comment:
I know it is detailed, but is it better this way?
```tsx
<Input type="text" />
```

Reply:
This is a fundamental mistake on my part. I will fix it.